common4j/src/main/com/microsoft/identity/common/java/providers/oauth2/OpenIdConnectPromptParameter.java

```diff
@@ -58,7 +58,15 @@
     /**
      * acquireToken will send prompt=consent to the authorize endpoint.  The user will be prompted to consent even if consent was granted before.
      */
-    CONSENT;
+    CONSENT,
+
+    /**
+     * acquireToken will send prompt=create to the /authorize endpoint.  The user will be prompted to create a new account.
+     * Requires configuring authority as type "AzureADMyOrg" with a tenant_id.
+     * <p>
+     * Prerequisite: https://docs.microsoft.com/en-us/azure/active-directory/external-identities/self-service-sign-up-user-flow
```

Comment:
What is the end user experience if the prerequisite is not met?

Reply:
From my testing, it sends the user to the login page, which doesn't have an option for creating an account.